src/devices/controller/controller.cpp

```diff
@@ -1,35 +1,57 @@
+h #include<Arduino.h>
 #include <FastLED.h>
+#include <arduino-timer.h>
+#include <median-filter.h>
 
-#include "Arduino.h"
 #include "FakeLedManager.hpp"
 #include "analog-button.h"
-#include "arduino-timer.h"
 #include "arduino/RadioHeadRadio.hpp"
+#include "buttons.h"
 #include "generic/NetworkManager.hpp"
 #include "generic/RadioStateMachine.hpp"
+#include "leds.h"
+
+    // Which mode of control the device is in
+    enum class ControllerMode {
```

Comment:
Why is this indented like this?

Reply:
Oops, there was a stray 'h'. Removed it, and fixed this formatting.